### ISSUE TITLE
Warn appear in multiple ui.plotly

### ISSUE THREAD

**Issue by Wzhipeng** [zauberzeug/nicegui#2435]
### Description

When my page calls many ui.plotly drawings, the following error will appear.
```
MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 plotly_animated listeners added. Use emitter.setMaxListeners() to increase limit
    at u (http://127.0.0.1:5500/_nicegui/1.4.11/libraries/a7b1c4d4fc8dfc15b2e1a1f5e04b13ae/plotly.min.js:8:2419952)
    at a.addListener (http://127.0.0.1:5500/_nicegui/1.4.11/libraries/a7b1c4d4fc8dfc15b2e1a1f5e04b13ae/plotly.min.js:8:2422536)
    at Proxy.update (http://127.0.0.1:5500/share/gAAAAABlsOaZuuvr0-D8W7QgcfJ4S5yd1BAtYOEUW-IQR7zyNP2GW5xU8lELhd6XtN6zQIl_ZeSLM5VHakN9PcRyKChrLCXeNcAnRVR5pYppKu_R-kRsoLgSVedfd55NZ_lm3WicliNvtvTrPWjvQ_k-cp8eSl68Eouc5DS_3XcGgUciOrx0wmacUVopTU2YuW3iXEMPQmYZ0Z0VbhpmN5zcMyI5vHF87J3gcSwoTvJRyHcPIzzczA8fKC-MwQj9KsY1rqW8ncHeLgterp8SPz1yxfDoMyyXl3r6WHzbhounoGXLzQv7fvLrnLVWGxHPcdLr64ndq4Fe2EfQ11mmPiBtxunx5LcOuLW9fFYgu5ueVZwWUoEDL5n-QbMThb6kLviAQOMlvNAC0kINtSa1RhEmWETc-sJzjw==:556:18)
    at Proxy.updated (http://127.0.0.1:5500/share/gAAAAABlsOaZuuvr0-D8W7QgcfJ4S5yd1BAtYOEUW-IQR7zyNP2GW5xU8lELhd6XtN6zQIl_ZeSLM5VHakN9PcRyKChrLCXeNcAnRVR5pYppKu_R-kRsoLgSVedfd55NZ_lm3WicliNvtvTrPWjvQ_k-cp8eSl68Eouc5DS_3XcGgUciOrx0wmacUVopTU2YuW3iXEMPQmYZ0Z0VbhpmN5zcMyI5vHF87J3gcSwoTvJRyHcPIzzczA8fKC-MwQj9KsY1rqW8ncHeLgterp8SPz1yxfDoMyyXl3r6WHzbhounoGXLzQv7fvLrnLVWGxHPcdLr64ndq4Fe2EfQ11mmPiBtxunx5LcOuLW9fFYgu5ueVZwWUoEDL5n-QbMThb6kLviAQOMlvNAC0kINtSa1RhEmWETc-sJzjw==:520:10)
    at http://127.0.0.1:5500/_nicegui/1.4.11/static/vue.global.prod.js:3:2763
    at Lt (http://127.0.0.1:5500/_nicegui/1.4.11/static/vue.global.prod.js:1:14656)
    at jt (http://127.0.0.1:5500/_nicegui/1.4.11/static/vue.global.prod.js:1:14734)
    at t.__weh.t.__weh (http://127.0.0.1:5500/_nicegui/1.4.11/static/vue.global.prod.js:3:2642)
    at nn (http://127.0.0.1:5500/_nicegui/1.4.11/static/vue.global.prod.js:1:15938)
    at sn (http://127.0.0.1:5500/_nicegui/1.4.11/static/vue.global.prod.js:1:16234)
```

**Comment by falkoschindler** [comment]
Hi @Wzhipeng,

Can you, please, try to provide a minimal reproduction for the warning?
And what OS, browser and Python version are you using?

**Comment by Wzhipeng** [comment]
Hi @falkoschindler  I am on mac 12.3 and use chrome browser， python version is 3.11.5

**Comment by Wzhipeng** [comment]
I think you can create 20 ui.plotly locally and see if this error occurs in the browser console

**Comment by falkoschindler** [comment]
Ah, ok, here is a minimal reproduction:

```py
import plotly.graph_objects as go

for _ in range(9):
    ui.plotly(go.Figure())
```

Strangely the warning occurs with 9 or more plots, even though the warning mentions "11 plotly_animated listeners".

**Comment by platinops** [comment]
Isn't this due to the events being created/forwarded by vue on each update of the chart?

https://github.com/zauberzeug/nicegui/blob/194a64a6f516579869bc4602ec6378ae99cfc04d/nicegui/elements/plotly.vue#L28-L62

I'm guessing this for loop should only run when the element is _mounted_, not when it is updated?

**Comment by platinops** [comment]
Should be fixed by #2505
